pkg/common/unittestcommon/types.go

```diff
@@ -43,7 +44,7 @@ type mockVolumeMigration struct {
 type MockVolumeMigrationService interface {
 	// GetVolumeID returns VolumeID for given VolumePath
 	// Returns an error if not able to retrieve VolumeID.
-	GetVolumeID(ctx context.Context, volumePath string) (string, error)
+	GetVolumeID(ctx context.Context, volumeSpec migration.VolumeSpec) (string, error)
```

Comment:
Nit: Update doc for method. GetVolumeID returns VolumeID for a given migration volumeSpec

Reply:
fixed.